src/Javis.jl

```diff
@@ -417,6 +429,10 @@ mutable struct InternalRotation <: InternalTransition
     center::Point
 end
 
+mutable struct InternalScaling <: InternalTransition
```

Comment:
You know, I have been thinking and it may be wise to actually explain each mutable struct that we define. The rationale for this is that if others were to hack on Javis, it may prove difficult to understand. Adding a small docstring for each one may be wise. Thoughts?

Reply:
We might want to have an actual development documentation for those as well but yeah as a start it might be good to have some docstrings.

Reply:
That makes sense. Maybe this could be a separate issue? Feel free to open an issue and resolve this convo